process/block/metablock.go

```diff
@@ -533,7 +528,13 @@ func (mp *metaProcessor) CommitBlock(
 
 	mp.indexBlock(header, tempHeaderPool)
 
-	go mp.displayMetaBlock(header)
+	go mp.headersCounter.calculateNumOfShardMBHeaders(header)
```

Comment:
I would remove go from this line to be sure that the displayLogInfo method would have the new calculated values in place at the execution time, or better, you can remove this line at all and put it inside displayLogInfo in the first line as you also have header parameter passed there

Reply:
Done.